### PR TITLE
=doc correct references to non-existent ShardRegion state messages / return types

### PR DESCRIPTION
**Pull request by timcharper**
As mentioned in gitter a few weeks ago:

:point_up: [April 15, 2016 4:20 PM](https://gitter.im/akka/dev?at=571169152c97111664326046)

### CODE REVIEW

File: akka-docs/rst/scala/cluster-sharding.rst [hunk 1]
```diff
@@ -361,11 +361,11 @@ Inspecting cluster sharding state
 ---------------------------------
 Two requests to inspect the cluster state are available:
 
-``ClusterShard.GetShardRegionState`` which will return a ``ClusterShard.ShardRegionState`` that contains
```
Comment (ktoso) [inline akka-docs/rst/scala/cluster-sharding.rst]: Nice catch, thanks!

**Comment by ktoso** [comment]
LGTM, thanks a lot!

**Comment by timcharper** [comment]
Oops. Java is wrong. I'll push another revision soon.

**Comment by timcharper** [comment]
Nevermind! I was viewing the patch from my phone and misread the diff. It's correct.

**Comment by johanandren** [comment]
LGTM

**Pull request merged**